### PR TITLE
(maint) Merge up 6.x into main

### PR DESCRIPTION
**Pull request by luchihoratiu**
Commits:
-  (packaging) Updating the puppet.pot file
-  (PUP-9176) List failed module install dependencies
-  (packaging) Updating manpage file for 6.x
-  (PUP-10795) logdest accepts multiple locations
-  (PUP-10725) add aliasing mechanism for settings

Conflicts:
-  lib/puppet/defaults.rb (in defaults.rb, the serverport setting hook from main was interfering with the new alias mechanism added in 6.x)

### CODE REVIEW

File: man/man5/puppet.conf.5 [hunk 1]
```diff
@@ -1105,7 +1105,7 @@ crit
 .IP "" 0
 .
 .SS "logdest"
-Where to send log messages\. Choose between \'syslog\' (the POSIX syslog service), \'eventlog\' (the Windows Event Log), \'console\', or the path to a log file\.
+Where to send log messages\. Choose between \'syslog\' (the POSIX syslog service), \'eventlog\' (the Windows Event Log), \'console\', or the path to a log file\. Multiple destinations can be set using a comma separated list (eg: \fB/path/file1,console,/path/file2\fR)
```
Comment (gimmyxd) [inline man/man5/puppet.conf.5:1108]: From what i remember man files conflicts should be ignored at merge-up as they will be autogenerated for each stream. Might be wrong about this

Comment (luchihoratiu) [reply man/man5/puppet.conf.5:1108]: I was confused about this also as there weren't conflicts regarding them so I left them be 😅

Comment (GabrielNagy) [reply man/man5/puppet.conf.5:1108]: The automatic merge driver ignores the man files, so they should be ignored on manual merge-ups as well.

File: lib/puppet/defaults.rb [hunk 2]
```diff
@@ -1319,25 +1319,16 @@ def self.initialize_default_settings!(settings)
...
-      :type       => :port,
-      :desc       => "The default port puppet subcommands use to communicate
-      with Puppet Server. (eg `puppet facts upload`, `puppet agent`). May be
-      overridden by more specific settings (see `ca_port`, `report_port`).",
-      :hook       => proc do |value|
-        Puppet[:masterport] = value unless Puppet.settings.set_by_config?(:masterport)
-      end
-    },
     :masterport => {
       :default    => "$serverport",
```
Comment (gimmyxd) [inline lib/puppet/defaults.rb:1323]: This is adding a circular dependency between `masterport` and `serverport`. Should be changed to `8140` like on 6.x: https://github.com/puppetlabs/puppet/blob/6.x/lib/puppet/defaults.rb#L1375

**Review by gimmyxd — APPROVED** [review]

**Pull request merged**